docs/user_guide/datasets/boston_housing_data.rst

```diff
@@ -402,47 +412,17 @@ different types of statistical analyses and different predicted outcomes.
 If you are searching for a house pricing dataset to use for benchmarking
 purposes or to create a hedonic pricing model, scikit-learn recommends the
 California housing dataset (:func:`sklearn.datasets.fetch_california_housing`)
-or the Ames housing dataset [#6]_
+or the Ames housing dataset :footcite:`scikitlearn2022ames`
 in place of the Boston housing dataset, as using these datasets should not
 generate the same fairness-related harms.
 We strongly discourage using the Boston Housing dataset for machine learning
 benchmarking purposes, and hope this article gives
 you pause about using it in the future.
 
-.. topic:: References:
-
-  .. [#1] David Harrison, Daniel Rubenfield, `"Hedonic Housing Prices and the Demand for Clean Air" <https://deepblue.lib.umich.edu/bitstream/handle/2027.42/22636/0000186.pdf?sequence=1&isAllowed=y>`_,
-      Journal of Environmental Economics and Management, 1978.
-
-
-  .. [#2] Ali Al Bataineh, Devinder Kaur, `"A Comparative Study of Different Curve Fitting Algorithms in Artificial Neural Network using Housing Dataset" <https://ieeexplore.ieee.org/abstract/document/8556738>`_,
-      IEEE, 2018.
-
-
-  .. [#3] Mohsen Shahhosseini, Guiping Hu, Hieu Pham, `"Optimizing Ensemble Weights for Machine Learning Models: A Case Study for Housing Price Prediction" <https://lib.dr.iastate.edu/cgi/viewcontent.cgi?article=1187&context=imse_conf>`_,
-      Industrial and Manufacturing Systems Engineering Conference Proceedings and Posters, 2019.
-
-
-  .. [#4] Michael E. Tipping , `"The Relevance Vector Machine" <https://proceedings.neurips.cc/paper/1999/file/f3144cefe89a60d6a1afaf7859c5076b-Paper.pdf>`_,
-      1999.
-
-  .. [#5] John F. Kain, John M. Quigley, `"Housing Markets and Racial Discrimination: A Microeconomic Analysis" <https://www.nber.org/books/kain75-1>`_,
-      National Bureau of Economic Research (NBER), 1975.
-
-  .. [#6] Scikit-Learn, `"The Ames housing dataset" <https://inria.github.io/scikit-learn-mooc/python_scripts/datasets_ames_housing.html>_`,
-      2021.
-
-  .. [#7] Scikit-Lego, `"Fairness" <https://scikit-lego.netlify.app/fairness.html>`_,
-      2019.
-
-  .. [#8] M Carlisle, `"racist data destruction?" <https://medium.com/@docintangible/racist-data-destruction-113e3eff54a8>`_,
-      Medium, 2019.
-
-  .. [#9] Marshall Hargrave, `"Hedonic Pricing" <https://www.investopedia.com/terms/h/hedonicpricing.asp>`_,
-      Investopedia, 2021.
+References
+----------
 
-  .. [#10] `"Metropolitan Areas", <https://www.census.gov/history/www/programs/geography/metropolitan_areas.html>`_,
-        United States Census Bureau.
+.. footbibliography::
 
-  .. [#11] Kinmberlé Crenshaw, Mapping the margins: Intersectionality, identity politics, and violence against women of color,
-      Stanford Law Review, 1993, 43(6), 1241-1299.
+.. [#1] Marshall Hargrave, `"Hedonic Pricing" <https://www.investopedia.com/terms/h/hedonicpricing.asp>`_,
```

Comment:
Why not just leave the link in the run of the text? I'm not sure why this is being drawn out as a specific citation.

Reply:
I must have been absent-minded... that's what I wanted to revert actually... I totally agree